apps/rush-lib/src/data/test/jsonFiles/rushWithLockVersion.json

```diff
@@ -0,0 +1,18 @@
+{
+  "npmVersion": "4.5.0",
+  "rushMinimumVersion": "2.5.0",
+  "nodeSupportedVersionRange": ">=6.9.0 <7.0.0",
+  "projectFolderMinDepth": 1,
+  "projectFolderMaxDepth": 99,
+
+  "versionPolicies": [
+    {
+      "policyName": "testPolicy1",
+      "baseType": "lockStepVersion",
```

Comment:
>"baseType": "lockStepVersion", [](start = 6, length = 30)

This should be called something like "publishingModel".  Personally I have trouble thinking of it as a "base type" being inherited by the type described here; it's more like a supported model that the person chooses, and then there will be model-specific configurations.  In the schema, the publishingModel choice will determine what other fields are supported for this object.